### DISCUSSION TITLE
Add parameter in getHtml function to remove css classes

### DISCUSSION THREAD

**Discussion by thomasmfinalcad** [stevengharris/MarkupEditor#199] [Ideas]
Hello @stevengharris,
I think it could be a good id to add a parameter into `getHtml` function to remove `extra` elements in html and only return html balises with content.
Morehover, when I try to convert the result into NSAttributedString and break line is added.

### Actual
```html
<ul>
    <li>
        <p class="editor-paragraph" dir="ltr">NO <u>SYjjjjjjjj<i>​hhryhhghgggjf</i></u></p>
    </li>
</ul>
```

### Expected

```html
<ul>
    <li>
        <p>NO <u>SYjjjjjjjj<i>​hhryhhghgggjf</i></u></p>
    </li>
</ul>
```

**Comment by stevengharris** [comment]
This discussion was moved to an [issue](https://github.com/stevengharris/MarkupEditor/issues/200)
